### PR TITLE
Taxonomy Extra props definition

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: src/record/recordUpdater.test.ts [hunk 1]
```diff
@@ -11,10 +11,54 @@ import { User } from '../auth'
...
+    nodePath,
+    value,
+    validationNodePath = params.nodePath,
+    expectedFieldValidation,
+    expectedValidationFieldsSize,
+  } = params
+  const nodeToUpdate = TestUtils.getNodeByPath({ survey, record, path: nodePath })
+
+  const nodeUpdated = { ...nodeToUpdate, value }
+  const recordUpdated = Records.addNode(nodeUpdated)(record)
```
Comment (ramirobg94) [inline src/record/recordUpdater.test.ts:39]: why did you add the node to the record before using the updateNode?

Comment (SteRiccio) [reply src/record/recordUpdater.test.ts]: fixed now

**Review by ramirobg94 — APPROVED** [review]
